docs/docs/architecture/state.md

```diff
@@ -1,4 +1,3 @@
-# State
```

Comment:
Why remove the heading?

Reply:
- It causes problems when you have more than one H1 on a page, which is not allowed.
- The Knowledge Layer removes them, on the whole, because ...
- The H1 header is set in the configuration nav, which you have to set in order for the docs to build, so adding it to the page also is duplication.
- Following on from that, if you have H1 in the page, and in the config, updating one usually means the other doesn't get updates, so they can easily become disconnected and this can get confusing.

Reply:
A developer may include it as a submission, but we would generally remove it when managing the content.